src/rio.cpp

```diff
@@ -94,6 +94,7 @@ static const rio rioBufferIO = {
     0,              /* current checksum */
     0,              /* flags */
     0,              /* bytes read or written */
+    0,              /* keys loaded */
```

Comment:
Again this isn't really tracking the total keys loaded (also the other places with this comment in the file)